Formula/aws-sso-cli.rb

```diff
@@ -0,0 +1,22 @@
+class AwsSsoCli < Formula
+  desc "Securely manage AWS API credentials using AWS SSO"
+  homepage "https://github.com/synfinatic/aws-sso-cli"
+  url "https://github.com/synfinatic/aws-sso-cli.git",
+      tag:      "v1.9.10",
+      revision: "11ec65288f78702a5e80078c04e0ef8b4ac7cb67"
+  license "GPL-3.0-only"
```

Comment:
we can also add head build support
```suggestion
  license "GPL-3.0-only"
  head "https://github.com/synfinatic/aws-sso-cli.git", branch: "main"
```